src/main/java/org/breedinginsight/model/Trait.java

```diff
@@ -59,9 +59,9 @@
 
```

Comment:
If this class is really ObservationVariable, it may make sense to move the brapi trait object properties to a Trait object property like is being done for method and scale. Could be done in another card.

Reply:
agreed. part of what I imagined would be included in the refactoring mentioned above.

Reply:
sounds good